app/models/Workers.php

```diff
@@ -115,6 +115,22 @@ public function getEmail(): string
         return $this->email;
     }
 
+    /**
+     * Returns multidimensional array with id and full name of all workers
+     * @return array
+     */
+    public function getWorkersWithFullName()
+    {
+        $fullNameWorkers = [];
+        $allWorkers = Workers::find(['order' => 'firstName, lastName'])->toArray();
+
+        foreach ($allWorkers as $worker) {
```

Comment:
you should get your query full serve and never needed to loop over it.
Looping over results should be made by views

Reply:
also remember that model does the queries.... manipulating it is part of views/service
in this case this should never be made... the data should be returned as it is to the views